src/components/Banner.tsx

```diff
@@ -22,6 +22,14 @@ export function Banner({ path }: { path: string }): JSX.Element {
                 The contents of this page will only be retained until <b>end of May 2024</b>. If you own this page,
                 please maintain an equivalent in the new{' '}
                 <a href="https://www.notion.so/sourcegraph">Notion-based handbook</a>.
+                <br />
+                <br />
+                <i>
+                    If you're a Sourcegraph teammate, you can set up a redirection to the corresponding Notion page by
+                    making a pull request against{' '}
+                    <a href="https://github.com/sourcegraph/handbook/blob/main/data/notion_migration.yml">this file</a>{' '}
+                    to match the handbook path with the new Notion page.
+                </i>
```

Comment:
Anyone can set up a redirection right? Additionally the previous phrase already has a "if you own this page" condition

```suggestion
                    You can set up a redirection to the corresponding Notion page by
                    making a pull request against{' '}
                    <a href="https://github.com/sourcegraph/handbook/blob/main/data/notion_migration.yml">this file</a>{' '}
                    to match the handbook path with the new Notion page.
                </i>
```

Reply:
(also my brain is weird but I find large chunks of italicized text hurts to read 😆 )